src/content/docs/it/getting-started.mdx

```diff
@@ -7,6 +7,11 @@ import TranslatorList from '~/components/TranslatorList.astro'
 import ContributorList from '~/components/ContributorList.astro'
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
+:::tip[Astro version 3 is here!]
```

Comment:
Shouldn't we translate also "Astro version 3 is here"?

Reply:
Definitely! I totally missed that.